weave/config.py

```diff
@@ -98,8 +98,14 @@ def get_index_column_names():
 
 def get_file_system():
     """Get the filesystem to be used for storing baskets."""
+    if "JUPYTERHUB_USER" in os.environ:
+        return s3fs.S3FileSystem(
+            client_kwargs={"endpoint_url": os.environ["S3_ENDPOINT"]}
+        )
     return s3fs.S3FileSystem(
```

Comment:
Maybe use an environment variable "DISABLE_SSL"

if "DISABLE_SSL" in os.environ: